src/functions.ts

```diff
@@ -73,52 +73,28 @@ const createFunctionFile = async (
   fnId: string,
   fnFilePath: string,
 ) => {
-  // Bundle File
-  let isImportMapPresent = false;
-  const importMapPath = `${options.workingDirectory}/import_map.json`;
-
-  try {
-    const { isFile } = await Deno.stat(importMapPath);
-    isImportMapPresent = isFile;
-  } catch (_e) {
-    isImportMapPresent = false;
-  }
+  const fnFileRelative = path.join("functions", `${fnId}.js`);
+  const fnBundledPath = path.join(options.outputDirectory, fnFileRelative);
 
-  let result;
   try {
-    result = await Deno.emit(fnFilePath, {
-      bundle: "module",
-      check: false,
-      importMapPath: isImportMapPresent ? importMapPath : undefined,
+    // call out to deno to handle bundling
+    const p = Deno.run({
+      cmd: [
+        "deno",
+        "bundle",
+        fnFilePath,
+        fnBundledPath,
+      ],
     });
-  } catch (e) {
-    console.log(
-      "This is likely due to the newest versions of Deno no longer supporting Deno.emit(). Please downgrade your version of Deno to 1.21.3",
-    );
-    throw new Error(e);
-  }
 
-  // Write FN File and sourcemap file
-  const fnFileRelative = path.join("functions", `${fnId}.js`);
-  const fnBundledPath = path.join(options.outputDirectory, fnFileRelative);
-  const fnSourcemapPath = path.join(
-    options.outputDirectory,
-    "functions",
-    `${fnId}.js.map`,
-  );
+    const status = await p.status();
+    if (status.code !== 0 || !status.success) {
+      throw new Error(`Error bundling function file: ${fnId}`);
```

Comment:
note: Throwing this error here will mean it's thrown twice since it gets re-thrown in the main catch block. Alongside the `console.log` this looks like
```
Error bundling function file: reverse
error: Uncaught (in promise) Error: Error bundling function file: reverse
      throw new Error(`Error bundling function file: ${fnId}`);
```

I don't think this is a problem, just calling out as something I noticed.